Calculate/Calculate/Calculator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Linq.Expressions;
+
+namespace Calculate
+{
+    public class Calculator
+    {
+        private static readonly Func<int, int, int> Add = (left, right) => left + right;
+        private static readonly Func<int, int, int> Subtract = (left, right) => left - right;
+        private static readonly Func<int, int, int> Multiply  = (left, right) => left * right;
+        private static readonly Func<int, int, int> Divide  = (left, right) => 
+        right.Equals(0) ? left / right : throw new ArgumentException("Cannot Divide by 0");
```

Comment:
```suggestion
        right.Equals(0) ? throw new ArgumentException("Cannot Divide by 0") : left / right;
```
You've got your inline if/else statement backward. This is causing a test failure.